api-pagination.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.require_paths = ['lib']
 
   s.add_development_dependency 'rspec', '~> 3.0'
-  s.add_development_dependency 'grape'
+  s.add_development_dependency 'grape', '>= 0.10.1'
```

Comment:
Actually should this be `'>= 0.10.0'`? Or did the breaking change come in 0.10.1?